### ISSUE TITLE
Add Gradle support

### ISSUE THREAD

**Issue by nlisker** [wildfly/wildfly-glow#86]
Currently only Maven plugins exist for simplifying the use of Glow. It would be helpful to have Gradle integrations as well since some projects use Gradle instead of Maven.

**Comment by jfdenise** [comment]
Yeah, you can download Glow CLI, run it to provision the server. That should integrate in Gradle, no need for a specific Gradle integration. 
If you could report what solution you ended with in this issue, that would help us identify possible improvements.
Thank-you!
JF
